api/room_resource/schema.py

```diff
@@ -55,17 +55,12 @@ def mutate(self, info, **kwargs):
             id=kwargs['resource_id'], state="active").first()
         if not exact_resource:
             raise GraphQLError('Resource with such id does not exist.')
-        exact_quantity = exact_resource.quantity
```

Comment:
@joshuaocero please shed some light on why you asked @KMaina to remove these inventory checks

Reply:
Yeah. This is a bit controversial even the fact that when creating a resource we don't include the quantity of the resource @joshuaocero

Reply:
@TeamoreA Can you rephrase your statement above? I don't understand what you're trying to pass across.

Reply:
@jesseinit it was a plus one on the comment above. Asking the reason behind removing the quantity in the resources table.

Reply:
@d-kahara @TeamoreA my view is that converge is not an inventory system so keeping inventory is out of scope. However, the product team has the final say so we'll be syncing with the product team to seek clarity. This PR cannot wait however as it is blocking our migration to production.